tests/compile-tests/fail/odd-capacity/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "odd-capacity"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.63.0"
```

Comment:
Side note: reminds me there's a proposal to have "rust scripts" - `.rs` files where `Cargo.toml` can be embedded (but not required). We could use it instead when it comes out.